network/src/counters.rs

```diff
@@ -171,6 +171,33 @@ pub fn rpc_bytes(
     ])
 }
 
+pub static INVALID_RPC_MESSAGES: Lazy<IntCounterVec> = Lazy::new(|| {
```

Comment:
this is outbound, self-initiated, right?

Reply:
Yes.